tests/testthat/test-getDescriptionStats.R

```diff
@@ -17,119 +17,119 @@ Loblolly$height_w_missing <- Loblolly$height
 Loblolly$height_w_missing[sample(1:nrow(Loblolly), size=4)] <- NA
 
 test_that("Check mean function",
-{
-  stats <- by(Loblolly$height, Loblolly$young, mean)
-  a <- getDescriptionStatsBy(Loblolly$height, Loblolly$young,
-                             statistics=TRUE,
-                             digits=2, statistics.sig_lim=10^-4)
-  # Check that it contains the true mean
-  expect_true(grepl(round(stats[["No"]], 2), a[1,"No"]),
-              info="Expected the mean")
-  expect_true(grepl(round(stats[["Yes"]], 2), a[1,"Yes"]),
-              info="Expected the mean")
-
-  # Check that it contains the sd
-  stats <- by(Loblolly$height, Loblolly$young, sd)
-  expect_true(grepl(round(stats[["No"]], 2), a[1,"No"]),
-              info="Expected the sd")
-  expect_true(grepl(round(stats[["Yes"]], 2), a[1,"Yes"]),
-              info="Expected the sd")
-
-  true_wilc_pv <- txtPval(wilcox.test(Loblolly$height ~ Loblolly$young)$p.value,
-                          statistics.sig_lim=10^-4)
-  expect_equal(as.character(a[1, "P-value"]),
-               true_wilc_pv)
-
-  # Check P-value without truncation
-  a <- getDescriptionStatsBy(Loblolly$height, Loblolly$age == 10,
-                             statistics=TRUE,
-                             digits=2, statistics.sig_lim=10^-4)
-  true_wilc_pv <- txtPval(wilcox.test(Loblolly$height ~ Loblolly$age == 10)$p.value,
-                          statistics.sig_lim=10^-4)
-  expect_equal(as.character(a[1, "P-value"]),
-               true_wilc_pv)
-})
+          {
```

Comment:
I'm using RStudio defaults, but it seems that it now auto-aligns everything, all the time. I'll fix it tonight.